servicecontrol/installation-powershell.md

```diff
@@ -108,16 +108,12 @@ Additional parameters for `Invoke-ServiceControlInstanceUpgrade` may be required
 
 ### Licensing
 
-Adding the license file to the registry can be done by running the following cmdlet. The license file is now machine wide and is available to be used by all instances of ServiceControl.
+Adding the license file to the registry can be done by running the following cmdlet. The license file is  is available to be used by all instances of ServiceControl. The license file is added to the `HKEY_LOCAL_MACHINE` registry hive so it available to be all instances of ServiceControl
```

Comment:
> The license file is is

double `is`. 

Also, that sentence can probably be removed entirely as the last sentence says the same thing anyway.

Also the last sentence contains the phrase:
> so it available to be all instances of

I recommend:

> Run the following cmdlet to add a license file to the registry.
> SNIPPET
> The license file is added to the `HKEY_LOCAL_MACHINE` registry hive so it is available to all instances of ServiceControl installed on the machine.